tensorboard/webapp/widgets/line_chart_v2/lib/worker/compact_data_series.ts

```diff
@@ -34,7 +34,7 @@ export function compactDataSeries(
     return len + data.points.length;
   }, 0);
   let seriesIndex = 0;
-  const flattenedSeries = new Float32Array(totalLength * 2);
+  const flattenedSeries = new Float64Array(totalLength * 2);
```

Comment:
It’s a bit suspicious to me that this change doesn’t need a change in
the types or documentation of `CompactDataSeries`. Could you add a brief
doc comment there indicating what `flattenedSeries` represents? i.e., a
buffer of packed f64s in some particular format?